cli/README.md

```diff
@@ -112,12 +114,16 @@ USAGE:
 
 FLAGS:
         --ffi               enables the FFI cheatcode
+        --force             force recompilation of the project, deletes the cache and artifacts folders
     -h, --help              Prints help information
     -j, --json              print the test results in json format
         --no-auto-detect    if set to true, skips auto-detecting solc and uses what is in the user's $PATH
     -V, --version           Prints version information
 
 OPTIONS:
+        --allow-failure <allow-failure>
+            if set to true, the process will exit with an exit code = 0, even if the tests fail [env:
+            FORGE_ALLOW_FAILURE=]
```

Comment:
this shouldn't require a value, 
this should be a simple flag where the presence of the flag is true, otherwise, false, could you check that?

Reply:
I don't know much about structopt, but it looks like it may not be possible to fallback to the env _and_ make it a boolean flag. https://issueexplorer.com/issue/TeXitoi/structopt/428